Extractor/ParamConverterExtractor.php

```diff
@@ -156,7 +156,10 @@ function ($converter) {
                     return false;
                 }
 
-                return $converter->getConverter() == "fos_rest.request_body";
+                return in_array(
+                    $converter->getConverter(),
+                    ["fos_rest.request_body", "kfz24.commons.json_schema.request_body"]
```

Comment:
Hmm it is a bit weird to have a "hard" dependency to  kfz24-commons here. If there is no other way i would say it is okay, but if it is possible to solve that without a hardcoded service-id from kfz24-commons here, i'd prefer that.